tools/cloud-build/babysit_tests.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python3
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Sequence, Dict, Tuple, Callable
+from dataclasses import dataclass
+from google.cloud.devtools import cloudbuild_v1
+from google.cloud.devtools.cloudbuild_v1.types.cloudbuild import Build, ApproveBuildRequest, ApprovalResult,RetryBuildRequest
+import time
+import argparse
+import subprocess
+
+Selector = Callable[[Build], bool]
+
+
+def selector_by_name(names) -> Selector:
+    def selector(build):
+        return any(trig_name(build) == n for n in names)
```

Comment:
Why is this set up to take multiple trigger names. Seems like the rest of the code only operates off a single name and it just adds additional complexity in these generated functions.

Reply:
It's a set up for group-selectors:
```
SELECTORS = {
  ...
  "slurm5": selector_by_name(["hpc-high-io-v5", "slurm-gcp-v5-hpc-centos7", ...])
```